docs/pages/access-controls/sso/oidc.mdx

```diff
@@ -1,30 +1,27 @@
 ---
-title: OAuth2 and OIDC authentication for SSH
-description: How to configure SSH access with OAuth2 or OIDC (OpenID connect) using Teleport
-h1: OAuth2 / OIDC Authentication for SSH
+title: OAuth2 and OIDC authentication
+description: How to configure Teleport access with OAuth2 or OIDC (OpenID connect)
```

Comment:
Isn't it more common to say "Open ID Connect (OIDC)"

Reply:
```suggestion
description: How to configure Teleport access with OAuth2 or OpenID connect (OIDC)
```